src/components/shared/DividendLineChart.tsx

```diff
@@ -100,13 +112,26 @@ const DividendLineChart: React.FC<DividendLineChartProps> = ({
         colors: textColor,
       },
     },
+    noData: {
+      text: 'No Data about Dividends is Found.',
+      align: 'center',
+      verticalAlign: 'middle',
+      offsetX: 0,
+      offsetY: 0,
```

Comment:
align: 'center',
verticalAlign: 'middle',
offsetX: 0,
offsetY: 0,
--------------
Is all unnecessary I think...